Update CHANGELOG

nywilken opened the pull request:
In moving to the shared HashiCorp release toolkit it was agreed to remove support for the following architectures ppc64le mips mips64 mipsle mipsle64 s390x since they are not part of the HashiCorp supported platforms. Nor are they available as a release binary for any of the Packer plugins.

This change adds an entry to the Packer CHANGELOG to call out the removal of the said architectures.

As we continue with the path of removing the plugins from Packer core supporting platforms outside of those managed by our tooling will be an issue for users as Packer needs the plugins to operate properly.

lbajolet-hashicorp reviewed (commented):
The change looks OK to me, however I have one question: is it normal that it's under the `1.8.0` release and not the current/next one? Was the removal of those architectures done with 1.8.0?

nywilken commented:
> 

Great question. The removal was in 1.8.0. I was on the fence of adding a note for the upcoming release to call out that the binaries were removed as of 1.8.0. Do you think that would be helpful?

lbajolet-hashicorp commented:
Just as a clarification this might be a good idea yes, though I'd assume anyone relying on any of these archs have probably seen that the builds were not available anymore.
Still, stating that we omitted specifying that when releasing a former version would make it clear that the changelog was amended post-publication.

nywilken commented:
@lbajolet-hashicorp could you give this another look over please. I added an entry for the upcoming release and updated the text a bit https://github.com/hashicorp/packer/pull/11934/commits/ceefc9f9c14cedcab48eb4d677cc47a731e7a57f

lbajolet-hashicorp reviewed (approved):
LGTM with the latest note, thanks!
